pkg/containerd/events.go

```diff
@@ -287,6 +401,45 @@ func createPortMappingFromString(portMapping string) (nat.PortMap, error) {
 	return portMap, nil
 }
 
+func extractIPAddress(line string) (string, error) {
+	// Regular expression pattern to match the IP address
+	ipPattern := `inet\s+(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\/\d{1,2}`
+
+	rx := regexp.MustCompile(ipPattern)
+
+	matches := rx.FindStringSubmatch(line)
+	segments := 2
+	if len(matches) < segments {
+		return "", ErrIPAddressNotFound
+	}
+
+	return matches[1], nil
+}
+
+// ported from:
+// https://github.com/containernetworking/plugins/blob/2b097c5a62dacfd2f9ea4268faa4fc04bd5343f5/pkg/utils/utils.go#L39
+// however module requires Go 1.20
```

Comment:
Is there a reason we don't just bump up the requirements?

Reply:
I did bump initially, however, that requires our environment to also get updated. Given the time crunch, I thought it's best to leave it for 1.10.